tests/test_pagination.py

```diff
@@ -97,7 +98,7 @@ def pagination_test(self):
                     location_id = [location['id'] for location in self.client.get_all('locations')][0]
                     for i in range(num_to_post):
                         new_objects.append(self.client.create_order(location_id))
-                elif self.API_LIMIT.get(stream) < self.BATCH_LIMIT: # not all streams have batch endpoints
+                elif self.API_LIMIT.get(stream) < self.BATCH_LIMIT or stream == 'inventories': # not all streams have batch endpoints
```

Comment:
I see how `elif self.API_LIMIT.get(stream) < self.BATCH_LIMIT:` worked before. But I think we should change this to be more explicit. The `BATCH_LIMIT` is for the Catalog Object streams, but I'm okay with reusing it since `inventories` also has a limit of 1000. But we have 3 groups of streams now: Orders, Not Catalog Object Streams, and Catalog Object Streams

I think it should look something like this
```
if stream == 'orders':
  pass
elif stream in {'a', 'b', 'c'}: # The not-catalog-object streams
  pass
elif stream in {'d', 'e', 'f'}: # The catalog object ones
  pass
else:        # I could live with or without this
  raise Error
```

Reply:
Good call I thinks that is more clear.

Reply:
Done